common/lib/Munin/Common/Config.pm

```diff
@@ -113,6 +113,8 @@ my %legal = map { $_ => 1 } qw(
 	sum
 	text
 	timeout
```

Comment:
Since you are introducing another *timeout* configuration, it would be good to amend the description of the current *timeout*. Users should get a clear understanding, which timeouts are used for which purpose.

Reply:
Agree, thought to do it, but unfortunately I'm not sure where this timeout comes in place :(

The only location I see it used is at https://github.com/munin-monitoring/munin/blob/stable-2.0/master/lib/Munin/Master/Node.pm#L91 which seems to be quite the same as the `process_timeout_worker` I introduced?

Maybe someone can make me understand this?

Reply:
I think, it was introduced in b8de4f8aae734e36b2220a07d6b763d86b3afe49, but handles *only* the connection timeout (maximum delay between trying to connect and establishing the connection). The time spent for further processing is not related to this setting.

Thus it would be great, if you clarify this in the description of the *timeout* setting in munin.conf.

Reply:
Ah thanks, I'll add it to the documentation!